runner/internal/report/report.go

```diff
@@ -9,14 +9,17 @@ import (
 	"strings"
 	"time"
 
+	"github.com/benchttp/engine/internal/cli/ansi"
```

Comment:
Nitpick: I'm not sure about this dependency.

`runner/internal/...` depends on `internal/cli/...`.

Yet `internal/cli` depends on `runner`.

It is ok because it **is** different packages. Still I find it strange because as `ansi` is nested in `cli`.

We may argue that it is strange that a core package (`report`) has io/visualization concerns.


Although, not deal breaker _at all_ for the pr.

Reply:
> We may argue that it is strange that a core package (report) has io/visualization concerns.

I agree, this is the root of the problem to me. It's ok having a `Report.String` but it should not be specific to a CLI with ansi codes.

Probably a good candidate for refactoring in #26